src/Elastic.Apm/Model/Transaction.cs

```diff
@@ -378,6 +383,9 @@ public void End()
 			_isEnded = true;
 			if (!isFirstEndCall) return;
 
+			var handler = Ended;
+			handler?.Invoke(this, EventArgs.Empty);
```

Comment:
Would it make sense to do `Ended = null` here? Just to make sure we don't hold a reference to anything longer than needed.

Reply:
yep, will add.